xla_native_functions.yaml

```diff
@@ -162,6 +162,7 @@ supported:
   - empty.SymInt
   - empty_strided
   - expand
+  - expand.SymInt
```

Comment:
hmm, this would probably break since we switched to a single overload for `expand`.